airflow/www/static/js/duration_chart.js

```diff
@@ -0,0 +1,35 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+function handleCheck() {
+  if ($('#isCumulative').is(':checked')) {
+    $('#dur_chart').hide();
+    $('#cum_dur_chart').show();
+  } else {
+    $('#dur_chart').show();
+    $('#cum_dur_chart').hide();
+  }
+}
+$(document).on('chartload', () => {
+  handleCheck();
+});
+
+$('#isCumulative').on('click', () => {
+  handleCheck();
+});
```

Comment:
Couldn't we do this?

```suggestion
$(document).on('chartload', handleCheck);
$('#isCumulative').on('click', handleCheck);
```